rosie/chamber_of_deputies/classifiers/election_expenses_classifier.py

```diff
@@ -14,11 +14,13 @@ class ElectionExpensesClassifier(TransformerMixin):
         Brazilian Federal Revenue category of companies, preceded by its code.
     """
 
-    def fit(self, X):
-        return self
+    def fit(self, dataframe):
+        pass
 
-    def transform(self, X=None):
-        return self
+    def transform(self, dataframe=None):
+        pass
 
-    def predict(self, X):
-        return X['legal_entity'] == '409-0 - CANDIDATO A CARGO POLITICO ELETIVO'
+    def predict(self, dataframe):
+        ELECTION_LEGAL_ENTITY = '409-0 - CANDIDATO A CARGO POLITICO ELETIVO'
```

Comment:
PEP8 says that [constants usually should be defined in the module level](https://www.python.org/dev/peps/pep-0008/#constants) (not method level), would you mind changing this minor detail?